docs/marketplace/user/intro-to-spryker-marketplace/merchant-onboarding.md

```diff
@@ -0,0 +1,149 @@
+---
+title:  Merchant onboarding
+description: This document describes the onboarding process for merchants and gives step-by-step instructions about completing it.
```

Comment:
change to: This document describes the onboarding process for merchants and gives step-by-step instructions for the marketplace operator about completing it.